src/sass/_permission_groups.scss

```diff
@@ -0,0 +1,152 @@
+.permission-groups {
+  .search-wrapper {
+    margin-bottom: -0.3rem; // needed to align button with search filter
+    width: 100%;
+  }
+
+  .groups-table {
+    .name {
+      width: 15%;
+    }
+
+    .description {
+      width: 45%;
+
+      span {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+
+        @supports (-webkit-line-clamp: 2) {
+          -webkit-box-orient: vertical;
+          display: -webkit-box;
+          -webkit-line-clamp: 2;
+          overflow: hidden;
+          text-overflow: ellipsis;
+          white-space: initial;
+          word-wrap: break-word;
+        }
+      }
+    }
+
+    .identities,
+    .permissions {
+      width: 10%;
+    }
+  }
+}
+
+.show-menu-above {
+  bottom: 1.8rem;
+}
+
+.group-identities-panel,
+.edit-permissions-panel {
+  .modified-actions {
+    align-items: center;
+    display: flex;
+    gap: $sph--large;
+    justify-content: flex-end;
+    margin-bottom: calc(0.75 * $spv--large);
+
+    .modified-status {
+      align-items: center;
+      display: flex;
+      gap: $sp-unit;
+
+      span {
+        color: $color-mid-dark;
+      }
+    }
+  }
+}
+
+#group-identities-table {
+  .selected-row {
+    background-color: white;
+  }
+
+  .modified-status {
+    width: 50px;
+  }
+}
+
+.delete-group-confirm-modal {
+  // remove the cancel button
```

Comment:
Might be more straight forward to use a bare `Modal` than the `ConfirmationModal` and then adjusting it to this custom use case?

Reply:
yes that makes sense :+1: